src/styles/block/card.scss

```
@@ -0,0 +1,68 @@
+.card {
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  padding: 32px 16px 16px;
+  width: 200px;
+  &__image {
```

add an empty line between selectors for better readability
```suggestion
  width: 200px;
  
  &__image {
```